tensorbay/opendataset/_utility/voc.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env python3
+#
+# Copyright 2021 Graviti. Licensed under MIT License.
+#
+
+"""_get_data function for VOC2012Detection dataset and UrbanObjectionDetection dataset."""
+
+import os
+from typing import Set
+
+from tensorbay.dataset import Data
+from tensorbay.label import LabeledBox2D
+
+try:
+    import xmltodict
+except ModuleNotFoundError:
+    from tensorbay.opendataset._utility.mocker import xmltodict  # pylint:disable=ungrouped-imports
+
+
+def _get_data(
```

Comment:
This function should not be private because it needs to be used outside this module.
And also it needs a better function name.